typescript/sdk/src/router/HyperlaneRouterDeployer.ts

```diff
@@ -21,40 +15,31 @@ export abstract class HyperlaneRouterDeployer<
   Config extends RouterConfig,
   Factories extends HyperlaneFactories,
 > extends HyperlaneDeployer<Config, Factories> {
-  constructor(
-    multiProvider: MultiProvider,
-    configMap: ChainMap<Config>,
-    factories: Factories,
-    options?: DeployerOptions,
-  ) {
-    super(multiProvider, configMap, factories, {
-      logger: debug('hyperlane:RouterDeployer'),
```

Comment:
Having the logger namespaced is nice, can we keep that?

Reply:
yeah though they're not actually namespaced